rest-api-model-ql/src/main/kotlin/org/modelix/sample/restapimodelql/ModelServerLightWrapper.kt

```diff
@@ -0,0 +1,145 @@
+package org.modelix.sample.restapimodelql
+
+import University.Schedule.*
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.websocket.*
+import jetbrains.mps.lang.core.N_BaseConcept
+import kotlinx.coroutines.delay
+import org.modelix.client.light.LightModelClient
+import org.modelix.client.light.WebsocketConnection
+import org.modelix.metamodel.typed
+import org.modelix.model.repositoryconcepts.N_Module
+import org.modelix.model.repositoryconcepts.N_Repository
+import org.modelix.model.repositoryconcepts.models
+import org.modelix.model.repositoryconcepts.rootNodes
+import org.modelix.model.server.api.buildModelQuery
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+
+val logger: Logger = LoggerFactory.getLogger("org.modelix.sample.restapimodelql.ModelServerLightWrapper")
+
+class ModelServerLightWrapper {
+
+    private var MPS_MODEL_NAME: String = ""
+    private var WS_CONNECTION: String
+    lateinit var globalModelClient: LightModelClient
+
+    constructor(host: String = "localhost", port: Int = 48302, models: String) {
+        GeneratedLanguages.registerAll()
+        WS_CONNECTION = "ws://$host:$port/ws"
+        MPS_MODEL_NAME = models
+    }
+
+    suspend fun createConnection() {
+
+        // we require a http client with WS support for the connection
+        logger.info("Connecting to light model-server at $WS_CONNECTION")
+        val modelClient = LightModelClient(WebsocketConnection(HttpClient(CIO) { install(WebSockets) }, WS_CONNECTION))
+        globalModelClient = modelClient
+        logger.info("Connection successful")
+
+        // the modelQL query to send
```

Comment:
Maybe some explanations on why exactly this query. This is probably important for readers.